.gitlab/deception.gitlab-ci.yml

```
@@ -0,0 +1,74 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
+
+hykkt-test-deception:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "a100_shared"
+    MY_CLUSTER: "deception"
+  extends:
+    - .pnnl_tags_template
+    - .hykkt_script_template  
+
+.report-status:
+  image: mrnonz/alpine-git-curl:alpine3.16
+  variables:
+    GIT_STRATEGY: none
+    STATUS_PROJECT: ORNL/hykkt
+    STATUS_NAME: CentOS-7-X86_64-gcc/9.1.0-cuda-11.4
```

```suggestion
    STATUS_NAME: (Deception) CentOS7/x86_64/gcc@9.1.0/cuda@11.4/cuda_arch=60,70,75,80
```
At least for now lets include a list of potential cuda architectures

---

Too long to have the whole status name to show up in github. Hence why I dropped the deception.

---

Looks like the character limit is somewhere between 45-48 characters.